Source/Core/DolphinQt2/DiscordHandler.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/DiscordHandler.h"
+
+#include "Common/Thread.h"
+
+#include "UICommon/DiscordPresence.h"
+
+DiscordHandler::DiscordHandler() : m_stop_requested(false)
```

Comment:
Common::Flag instances initialize to false by default, so the initializer list isn't necessary. This is sufficient as:

```cpp
DiscordHandler::DiscordHandler() = default;
```